lib/cloudant.js

```diff
@@ -203,7 +189,9 @@ Cloudant.prototype.create = function(model, data, options, cb) {
  *
  * @param {String} model The model name
  * @param {Object} data The model data
+ * @param {Object} options The options object
  * @param {Function} [callback] The callback function
+ * @returns {Function} [create] model create function
```

Comment:
I would probably put what get returned in `_insert` function then. Just a nitpick, feel free to do it in your way :p